raster-foundry/blasterjs
# Update simple form components to new theme system

## Target Branch
Note this is a request to merge to `feature/theme`, not `develop`. All theme-related PRs should target `feature/theme` until the theme upgrade is complete.

## Overview

Updates the following components to the new theme system:

- Label
- TextInput
- ColorInput
- FileInput
- TextArea
- Checkbox
- Radio
- Select

### Checklist


### Upgrade instructions

If there are any of the following in this PR, provide proper instructions on how to upgrade:


`size` prop removed from `Label` and `TextInput`

### Notes

This PR does **not** update `Field` and `ToggleField`.

## Testing Instructions

- Use docz pages and/or testbed app to confirm components work as expected

Closes #162 
Closes #165 
Closes #159 
Closes #161 
Closes #166 
Closes #158 
Closes #163 
Closes #164
